CONTRIBUTING.md

```diff
@@ -328,20 +345,29 @@ Resolves issue #123"
 
     ```bash
     # Squash multiple commits into one
-    git rebase -i HEAD~[number of commits to squash]
+    git rebase -i HEAD~{number of commits to squash}
+    # and follow the instructions in the editor to squash your commits
     # or squash all commits since branching from main
     git fetch
-    git rebase -i main
+    git rebase -i origin/main
     ```
 
     > [!NOTE]
     > This repository maintains a [linear commit history](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches#require-linear-history).
     >
-    > Always use [`rebase`](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) instead of `merge` when keeping your branch up to date with the `main` branch.
+    > Always use [`rebase`](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) instead of `merge` when keeping your branch up to date with the `main` branch (see previous step).
+
+1. **Merge the PR**: Once approved and all status checks have passed, including the branch being up to date with main, you can trigger a `fast-forward` merge by adding the `fast-forward` label to the pull request. This will initiate an automated, permission-checked `fast-forward` merge process. Only users with `write` or `admin` permissions on the repository can trigger this action. If you're an external contributor, a maintainer may need to do this for you.
```

Comment:
A tiny thing but I notice that the other numbered items use specific numbers rather than all using `1.`. My preference is to consistently use `1.` to avoid causing confusion (since specifying the numbers doesn't have the effect you may expect). How do you feel about changing all these to `1.`?

Reply:
Also, suggest small rewording for clarity, assuming this is correct: "...a maintainer may need to do this for you, as the automated process only responds to this tag when it has been added by an authorised user."